packages/v4/src/content/design-guidelines/usage-and-behavior/lists-and-tables/lists-and-tables.md

```diff
@@ -145,12 +151,30 @@ If a table contains these two attributes: (System Name | Last Sync) you may want
 
 If a table contains these three attributes: (System Name | Last Sync | Severity) you may want to show the system with the highest Severity because that is the system the user should tend to first.
 
+## When to use compact vs. basic spacing
+
+<img src="./img/compact-vs-basic-spacing.png"  alt="Compact vs. basic spacing"  width="1237"/> 
```

Comment:
I wonder if this should be 2 images, with markdown headings (that are linkable) for both "compact" and "basic". That would match the rest of the page, and make the "compact" and "basic" links I've left comments about make a little more sense if you wanted to link them individually.

Reply:
That's a great idea! Thanks.